tests/pipelines/text_to_video/test_text_to_video.py

```diff
@@ -170,36 +170,42 @@ def test_progress_bar(self):
 
 @slow
 @skip_mps
+@require_torch_gpu
 class TextToVideoSDPipelineSlowTests(unittest.TestCase):
-    def test_full_model(self):
+    def test_two_step_model(self):
         expected_video = load_numpy(
-            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/text_to_video/video.npy"
+            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/text_to_video/video_2step.npy"
         )
 
         pipe = TextToVideoSDPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b")
-        pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
-        pipe = pipe.to("cuda")
+        pipe = pipe.to(torch_device)
 
         prompt = "Spiderman is surfing"
         generator = torch.Generator(device="cpu").manual_seed(0)
 
-        video_frames = pipe(prompt, generator=generator, num_inference_steps=25, output_type="pt").frames
+        video_frames = pipe(prompt, generator=generator, num_inference_steps=2, output_type="pt").frames
         video = video_frames.cpu().numpy()
 
         assert np.abs(expected_video - video).mean() < 5e-2
 
-    def test_two_step_model(self):
+
+@nightly
+@skip_mps
+@require_torch_gpu
+class TextToVideoSDPipelineNightlyTests(unittest.TestCase):
+    def test_full_model(self):
         expected_video = load_numpy(
-            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/text_to_video/video_2step.npy"
+            "https://huggingface.co/datasets/hf-internal-testing/diffusers-images/resolve/main/text_to_video/video.npy"
         )
 
         pipe = TextToVideoSDPipeline.from_pretrained("damo-vilab/text-to-video-ms-1.7b")
-        pipe = pipe.to("cuda")
+        pipe.scheduler = DPMSolverMultistepScheduler.from_config(pipe.scheduler.config)
+        pipe = pipe.to(torch_device)
 
         prompt = "Spiderman is surfing"
         generator = torch.Generator(device="cpu").manual_seed(0)
 
-        video_frames = pipe(prompt, generator=generator, num_inference_steps=2, output_type="pt").frames
+        video_frames = pipe(prompt, generator=generator, num_inference_steps=25, output_type="pt").frames
         video = video_frames.cpu().numpy()
 
         assert np.abs(expected_video - video).mean() < 5e-2
```

Comment:
If the two step model is passing the assertion, I think it's okay to remove it from nightly unless we have a strong reason to.

Reply:
Remove the full and keep the two steps.